Support/Scripts/run-lldb-tests.sh

```diff
@@ -65,17 +71,19 @@ while test $# -gt 0; do
     --strace) opt_strace=true;;
     --use-lldb-server) opt_use_lldb_server=true;;
     --lldb-tests) shift
-	    LLDB_TESTS="$1";;
+      LLDB_TESTS="$1";;
     --target) shift
-	    TARGET="$1";;
-    --platform) shift
-	    PLATFORM="$1";;
+      TARGET="$1";;
+    --platform) PLATFORM=1;;
+    --pudb) PUDB="-m pudb";;
```

Comment:
Instead of storing this as a list of flags, I’d probably have opt_pudb=true, and check the conditional later when building the “args” array.

Reply:
It can't be to the args array, it has to be called before the python file name. `python -m module args-for-the-module`.